src/python/pants/notes/2.18.x.md

```diff
@@ -1,7 +1,202 @@
 # 2.18.x Release Series
 
+❗❗❗ This project is no longer being released via PyPI, and is instead released via GitHub Releases. See https://github.com/pantsbuild/pants/releases
+
+Individuals and companies can now [sponsor Pants financially](https://www.pantsbuild.org/docs/sponsorship).
+
+Pants is an open-source project that is not owned or controlled by any one company or organization, and does incur some expenses. These expenses are managed by Pants Build, a non-profit that was established for this purpose. This non-profit's only source of revenue is sponsorship by individuals and companies that use Pants.
+
+We offer [formal sponsorship tiers for companies](https://www.pantsbuild.org/docs/sponsorship), as well as individual sponsorships via [GitHub](https://github.com/sponsors/pantsbuild).
+
 ## What's New
 
+### Highlights
+
+- The improvements to AWS Lambda and Google Cloud Function support in 2.17 have continued in 2.18, including simpler configuration and support for AWS Lambda Layers.
+- Many more options can now be controlled when building Docker images.
+- Significant feature work and bug fixes to "deploy" backends like Helm and Terraform.
```

Comment:
This is just my very quick summary of the "most major" features based on a very approximate estimate of how fundamental the changes are and how often people ask about them. Definitely up for debate!